uui/components/tables/columnsConfigurationModal/ColumnsConfigurationModal.tsx

```diff
@@ -73,15 +57,15 @@ export function ColumnsConfigurationModal<TItem, TId, TFilter>(props: ColumnsCon
                 {!!amountPinned && (
                     <FlexRow cx={ styles.groupItems } size="30">
                         {groupedColumns.displayedPinned.map((c) => (
-                            <ColumnRow column={ c } key={ c.key } />
+                            <ColumnRow column={ c } key={ c.key } renderItem={ props.renderItem ? () => props.renderItem(c) : null } />
```

Comment:
why do we have `props.renderItem ? () => props.renderItem(c) : null` condition here, if we have the same inside `ColumnRow`?